### PR TITLE
Ft add atlabs module

### PR DESCRIPTION
**Pull request by addempsea**

### CODE REVIEW

File: lib/atlabs/README.md [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+const { atlabs } = require("innovation-sandbox");
+```
+
+## Common Credentials
+
+Below is a list of required credentials.
+
+### sandbox_key
+
+This can be found in the innovation sandbox dashboard after sign up. However `0ae0db703c04119b3db7a03d7f854c13` can be used for testing purposes.
```
Comment (akinjide) [inline lib/atlabs/README.md]: Ensure this documentation is consistent with changes here #17

File: lib/atlabs/SMS.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            path: '/atlabs/messaging',
+            credentials: { ...data },
+            payload: data.payload,
+        });
+    } catch (error) {
+        return error;
+    }
+};
+
+exports.PremiumSub = async data => {
```
Comment (akinjide) [inline lib/atlabs/SMS.js]: Avoid abbreviations, PremiumSub should be PremiumSubscription, apply the change to other method names below

File: package.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
   "description": "Node.js package for Innovation Sandbox",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint && mocha --timeout 30000",
+    "test": "npm run lint && mocha test/**/*.js --timeout 30000",
```
Comment (akinjide) [inline package.json]: mocha reads test/ directory automatically, change not required

File: test/atlabs/SMS.test.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
+const chai = require("chai");
+const nock = require("nock");
+
+const SMS = require("../../lib/atlabs/SMS");
+
+const scope = nock("https://sandboxapi.fsi.ng");
+const SMSMock = require("../fixtures/atlabs/SMS");
```
Comment (akinjide) [inline test/atlabs/SMS.test.js]: rename folder `fixtures` to `mock`

File: test/atlabs/SMS.test.js [hunk 5]
```diff
@@ -0,0 +1,104 @@
+const chai = require("chai");
+const nock = require("nock");
+
+const SMS = require("../../lib/atlabs/SMS");
+
+const scope = nock("https://sandboxapi.fsi.ng");
```
Comment (akinjide) [inline test/atlabs/SMS.test.js]: Use random URL for nock `i.e. faker.internet.url()`

File: test/fixtures/atlabs/Voice.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    callTo: `+234${faker.random.number(1000000)}`
+  },
+
+  FetchQueuedCall: {
+    phoneNumbers: `+234${faker.random.number(1000000)}`
+  },
+
+  UploadMediaFile: {
+    phoneNumber: `+234${faker.random.number(1000000)}`,
+    url: "https://sandbox.fsi.ng/images/svg/fsi-logo-dark.svg"
```
Comment (akinjide) [inline test/fixtures/atlabs/Voice.js]: Avoid using actual URL use random

File: lib/atlabs/SMS.js (hunk 2 above)
Comment (addempsea) [reply lib/atlabs/SMS.js]: Edited everywhere

File: package.json (hunk 3 above)
Comment (addempsea) [reply package.json]: Used recursive flag instead.

File: test/atlabs/SMS.test.js (hunk 4 above)
Comment (addempsea) [reply test/atlabs/SMS.test.js]: Renamed.

File: test/atlabs/SMS.test.js (hunk 5 above)
Comment (addempsea) [reply test/atlabs/SMS.test.js]: Done.

File: test/fixtures/atlabs/Voice.js (hunk 6 above)
Comment (addempsea) [reply test/fixtures/atlabs/Voice.js]: Changed.

**Pull request merged**